modules/local/gtdbtk/main.nf

```diff
@@ -1,16 +1,17 @@
 process GTDBTK {
 
     container 'quay.io/microbiome-informatics/gtdb-tk:2.1.0'
-    containerOptions "--bind ${gtdbtk_refdata}:/opt/gtdbtk_refdata"
 
-    label 'process_bigmem'
+    containerOptions "${ workflow.containerEngine == 'singularity' ?
+        "--bind ${gtdbtk_refdata}:/opt/gtdbtk_refdata":
+        "-v ${gtdbtk_refdata}:/opt/gtdbtk_refdata" }"
 
     input:
     path(genomes_fna, stageAs: "genomes_dir/*")
-    path gtdbtk_refdata
+    path(gtdbtk_refdata)
 
     output:
-    path("Taxonomy")
+    path 'gtdbtk_results.tar.gz', emit: gtdbtk_output_tarball
```

Comment:
That should require a fix in MAG_Uploader to support archive (I don't see GTDB-Tk result in test results folder)

Reply:
OK. I'll collect the bits that will require us to change on the MAG uploader in a ticket